sdcm/nemesis.py

```diff
@@ -3835,6 +3832,13 @@ def remove_node():
         # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
         self.nodetool_cleanup_on_all_nodes_parallel()
 
+        # full cluster repair
+        up_normal_nodes = self.cluster.get_nodes_up_and_normal(verification_node)
+        # Repairing will result in a best effort repair due to the terminated node,
+        # and as a result requires ignoring repair errors
+        with DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR, line="failed to repair"):
+            self.run_repair_on_nodes(nodes=up_normal_nodes, ignore_down_hosts=True)
+
```

Comment:
I thinks it should be done before adding new node to cluster, right afte rnode was removed.